docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+s# HVAC_SystemZoneAssignment - Rule 11-10
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** "The service water heating system type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2"  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each ServiceWaterHeatingEquipment  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW Equipment is applicable
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_equip_dict = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)`
+- look through each of the SHW bats in the building: `for shw_bat in shw_bats_and_equip_dict:`
+  - look through each SHW Equipment: `for service_water_heating_equipment_id in shw_bats_and_equip_dict[shw_bat]: CONTINUE TO RULE LOGIC`
+
+
+    ## Rule Logic:
+  - create a value to track the rule status: `rule_status = "FAIL"`
+  - create a variable to track any notes: `rule_note = """`
+  - we need to find the following information about the SWH:
+    - swh_type - (ELECTRIC_RESISTANCE_INSTANTANEOUS, ELECTRIC_RESISTANCE_STORAGE, GAS_STORAGE, OTHER)
+    - swh_type - one of the SWH BATs
```

Comment:
same variable name as line above?

Reply:
removed other variable as it wasn't being used

Reply:
Is "one of the SWH BATs" an inaccurate label. It does not appear correct.